appcues/src/main/java/com/appcues/ui/modal/DialogModal.kt

```diff
@@ -33,48 +43,119 @@ private val MAX_WIDTH_EXPANDED_DP = 560.dp
 private val MAX_HEIGHT_COMPACT_DP = Dp.Unspecified
 private val MAX_HEIGHT_MEDIUM_DP = 800.dp
 private val MAX_HEIGHT_EXPANDED_DP = 900.dp
-private const val SCREEN_PADDING = 0.05
+
+private val SCREEN_HORIZONTAL_PADDING = 12.dp
+private val SCREEN_VERTICAL_PADDING = 24.dp
```

Comment:
This is up for debate, but in tooltips we have this standard padding defined, and in modals we had a screen % padding factor being applied. This seemed inconsistent to me, and on the vertical padding in particular it could lead to slideouts seeming to be a fair bit further away from the top or bottom edge than expected. This also would affect top or bottom positioned dialog modals, which similarly seemed a bit too far from the edge, IMO. It is a matter of visual design though, and open to whatever product and design think is best if there are concerns. It felt best to be consistent with tooltips.

Reply:
SCREEN_PADDING was one of first things we did here (this is old code), it is ok to change from that to dp paddings, only question regarding tablet and landscape, which i recall was the reason why I went with % in the first place, but I guess since then we did specific layout for those cases as well. 
it should be well covered.

Reply:
right, I believe those are covered by the max width that gets set, but I will so review those to test - good call.

I did run the UI tests, and things work out fine in general - but there will be several image updates where the dialog gets updated to 12.dp margin left/right instead of something slightly larger than that in the previous code.

Reply:
im ok with that.

Reply:
I am second guessing this change now and think I'll put it back the way it was. On the 390dp width test device, for example, the padding on the sides is now 12 instead of about 20. It just looks like it's a little to horizontally stretched for a dialog modal. Maybe we could go with fixed vertical padding but keep horizontal as it was before.

Reply:
do you have a visual example of that?

Reply:
here is an example on pixel 6

|  before (and current)  | experimental 12 dp margin |
| ---- | ---- |
| ![Screenshot 2023-09-12 at 9 29 26 AM](https://github.com/appcues/appcues-android-sdk/assets/19266448/5e8de251-6b65-43ef-b0d0-235376e8d159) | ![Screenshot 2023-09-12 at 10 30 36 AM](https://github.com/appcues/appcues-android-sdk/assets/19266448/a13eb05e-2ed1-4260-af58-a26c01e0c75d) |

Reply:
Here is the "downside" to that - on a larger screen, a slideout along an edge appears a bit further from that edge than I would expect.  So maybe we should have different rules for normal dialogs (horizontal center) versus items with leading/trailing edge alignment?

![Screenshot 2023-09-12 at 10 40 36 AM](https://github.com/appcues/appcues-android-sdk/assets/19266448/d29b3b6e-23eb-4cc0-89d4-aeae37e593cf)

Reply:
maybe its just a matter of figuring out a good value for horizontal margins there, maybe 12 is good for tooltips but not good for modals, how is IOS dealing with this? is this one of those settings coming from the SwiftUI Package?

Reply:
maybe instead of 12. maybe 16 or 20, 24  may look the same on regular devices, and then should be good for mini modals too.